packs/classfeatures/blessed-swiftness.json

```diff
@@ -30,6 +30,24 @@
                 "selector": "speed",
                 "type": "status",
                 "value": 5
+            },
+            {
+                "effects": [
+                    {
+                        "affects": "allies",
+                        "includesSelf": false,
+                        "uuid": "Compendium.pf2e.feat-effects.Item.Effect: Blessed Swiftness"
+                    }
+                ],
+                "key": "Aura",
+                "predicate": [
+                    "champions-aura",
+                    {
+                        "not": "self:condition:unconscious"
+                    }
+                ],
+                "radius": "@actor.flags.pf2e.champion.aura",
+                "slug": "champions-aura"
```

Comment:
Do you want the same slug as Champion's Aura?

Reply:
Yes, they are supposed to merge